Pyspark/regular_patients/main.py

```diff
@@ -0,0 +1,45 @@
+'''This module implements the PatientsData class to analyze the regular patient data'''
+from regular_patients.logger import Log4J
+from regular_patients import PatientsData
+
+if __name__ == '__main__':
+
+    #solo para probar
+    import os
+    print("Current Working Directory:", os.getcwd())
+
+    FILE_PATH = "file:///Pyspark/regular_patients/input/enroll.csv"
```

Comment:
Try to avoid using harcoded path, especially when you need to make your code flexible to run on different OS. For example, it failed for me on Ubuntu and i needed to update path manually.  Try to use pathlib for your paths variables https://realpython.com/python-pathlib/